src/tests/ftest/soak/soak.yaml

```diff
@@ -1,146 +1,167 @@
 hosts:
-   test_machines:
-        daos_servers:
-            - boro-A
-        # either compute or partition can be used, not both
-        #compute:
-        #    - boro-A
-        partition: tp
-        gateway: []
-timeout: 1000
+    test_servers:
+        - boro-A
+        - boro-B
+        - boro-C
+        - boro-D
+    test_clients:
+        - boro-E
+        - boro-F
+        - boro-G
+        - boro-H
+        - boro-I
+        - boro-J
+        - boro-K
+        - boro-L
+timeout: 7200
+logdir: /tmp/soak
 server_config:
     name: daos_server
-
-pool1:
-    createmode:
-        mode_RW:
-             mode: 146
-    createset:
-        setname: daos_server
-    createsize:
-        size: 10000000000
-    createsvc:
-        svcn: 1
-
+#slurm partition names
```

Comment:
Should this be located with the hosts and it should be clear that you are using either test_servers/test_clients or partitions but not both?  Comments would help someone customizing this.

Reply:
With the split between client/server do the partition parameters need to change?